minutes/2024/240404Minutes.md

```diff
@@ -0,0 +1,55 @@
+# Minutes for April 4th, 2024
+
+## Attendees
+- Martino Sorbaro (Secretary)
+- Samantha Ahern
+- Valentina Hurtado-McCormick
+
+## Agenda
+- State of the Matlab lesson
+- Idea for community discussions
+- Outcome of the blog post
+- Outcome of the collaborations workshop
+- Status of new maintainers recruitment
+
+## Notes
+**Action items are indicated with @.**
+
+### State of the Matlab lesson
+Sam: the Matlab part works, but needs testing on Octave. There is no nice explanation on indexing.
+We briefly discuss whether this lesson could be sent back to the Carpentries Lab, but we decide to consider pulling lessons from the Lab to Software Carpentry instead (see below).
+
+### Ideas for community discussions
+- What kinds of things would people want to see as new lessons? First we need to come up with proposals.
+- What is preventing instructors from participating more in lesson development?
+
+### Outcome of the blog post
+We find that there was substantially no response from the community.
+
```

Comment:
We might be part of the rule rather than the exception. We tried something similar to get feedback on the shell lesson and did not get a single response/reaction whatsoever.